functions/text-transform/1.0/index.js

```diff
@@ -0,0 +1,34 @@
+const textTransform = async ({ value, transformation }) => {
+  let result;
+
+  switch (transformation) {
+    case 'downcase':
+      result = value.toLowerCase();
+      break;
+    case 'upcase':
+      result = value.toUpperCase();
+      break;
+    case 'capitalize':
+      result = capitalized(value);
+      break;
+    case 'parameterize':
+      result = parameterized(value);
+      break;
+    default:
+      result = value;
+  }
+
+  return { result };
+};
+
+const capitalized = (value) =>
```

Comment:
Can you place the capitalized and parameterized functions above the switch case. Always place a function above the call

Reply:
Fixed.